app.rb

```diff
@@ -0,0 +1,37 @@
+require 'sinatra/base'
+require './lib/chitter.rb'
+require './lib/database_connection_setup.rb'
+
+class ChitterApp < Sinatra::Base
+  # enable :sessions
+  # configure(:development) { set :session_secret, "something" }
+  get '/' do
+    redirect '/chitter'
+  end
+
+  get '/chitter' do
+    @peeps = Chitter.all
+    erb(:'chitter/index')
+  end
+
+  get '/chitter/new' do
+    erb(:'chitter/new')
+  end
+
+  post '/chitter' do
```

Comment:
This breaks your app, forces me to use a test database locally. Refactor to use database connection class